src/components/CookieConsent.astro

```diff
@@ -0,0 +1,157 @@
+<div class="consent-banner">
+  <div class="consent-info">
+    <p>Can we store cookies?</p>
+    <p>
+      By clicking “Accept”, you agree to the storing of cookies on your device
+      to analyse site usage.
+    </p>
+  </div>
+  <div class="consent-controls">
+    <a href="/privacy">Privacy Policy</a>
+    <button class="button">Reject</button>
```

Comment:
there's a cta-link global class we can reuse here, maybe turn it into cta-button to avoid confusions. it is used for the button on the hero image and was used for the button for the CFP.